assignment.md

```diff
@@ -14,4 +14,11 @@ Steps:
 
 The output of this assignment should be the command needed to pull and run the necessary image.
 
-Output: `<output goes here>`
\ No newline at end of file
+Output: `<output goes here>`
+`<docker pull hello-world>`
+`<docker run hello-world>`
+`<docker rmi hello-world>`
+`<docker images -a>`
+`<docker system prune>`
```

Comment:
FWIW, this is how I used to do it when I was first learning because who has time to remove single containers/images.  :)